src/plugins/transform-module/dynamic-import.ts

```diff
@@ -23,7 +21,7 @@ const requireInterop = (
   return t.callExpression(t.memberExpression(exp, t.identifier("then")), [
     t.arrowFunctionExpression(
       [t.identifier("m")],
-      t.callExpression(file.addHelper("interopRequireWildcard"), [
+      t.callExpression(file.hub.addHelper("interopRequireWildcard"), [
```

Comment:
(TODO: this is a runtime change need to check)

Reply:
I checked it against the source code from babel, the outcome is the same.

Reply:
I can change these to `as` type assertions if you want.